torch/_functorch/aot_autograd.py

```diff
@@ -736,13 +735,18 @@ def aot_function(
         partition_fn (Callable): A Python function that takes a joint forward
             and backward graph, and partitions it into separate forward and
             backward graphs.
-        decompositions (Dict): A dictionary to define the decomposition of
+        decompositions (Optional[Dict]): A dictionary to define the decomposition of
             larger Aten ops into simpler or core Aten ops.
+        num_params_buffers (int): Number of params and buffers in the forward graph. Default: 0.
+        keep_inference_input_mutations (bool): Whether to keep mutations to the input
+            tensors in the inference graph. Default: ``False``.
```

Comment:
I think these descriptions may be wrong, can someone help me correct them? Thanks so much!